Modules/Core/SpatialObjects/include/itkTubeSpatialObject.h

```diff
@@ -83,6 +83,13 @@ class ITK_TEMPLATE_EXPORT TubeSpatialObject : public PointBasedSpatialObject<TDi
   bool
   ComputeTangentsAndNormals();
 
+  /** Calculate the normalized tangent - Old spelling of function name */
+  bool
```

Comment:
Maybe it should be wrapped in legacy macro?

Reply:
Done!  Great idea!